spec/yarn.lock

```diff
@@ -303,7 +302,7 @@ dashdash@^1.12.0:
     safe-buffer "^5.1.1"
     xml2js "^0.4.17"
   optionalDependencies:
-    abstract-socket "^2.0.0"
+    abstract-socket "github:saghul/node-abstractsocket#35b1b1491fabc04899bde5be3428abf5cf9cd528"
```

Comment:
is this related?

Reply:
Uh, no.  I just ran `yarn`.  Not sure why this wasn't accurate before. I'll double check nothing nefarious is afoot

Reply:
Yeah this checks out, not sure why this wasn't updated before